tests/unit/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal.spec.jsx

```diff
@@ -300,9 +300,82 @@ describe('AddProfessionalCategoryModal Subject Updates', () => {
           defaultValue: subject.name
         })
   
-        const subjectElement = professionalSubjects[index];
+        const subjectElement = professionalSubjects[index]
         expect(subjectElement.value).toMatch(new RegExp(translatedSubjectName, 'i'))
       })
     })
   })
 })
+describe('AddProfessionalCategoryModal when clearing categories and subjects', () => {
+  let categoryAutocomplete
+  let professionalSubjects
+  let submitButton
+
+  beforeEach(async () => {
+    await waitFor(() => {
+      renderWithProviders(
+        <AddProfessionalCategoryModal
+          blockedCategoriesOptions={mockedBlockedCategory}
+          closeModal={mockCloseModal}
+          initialValues={initialValues}
+          isEdit
+        />
+      )
+    })
+    categoryAutocomplete = screen.getByLabelText(
+      /editProfilePage.profile.professionalTab.mainStudyCategory/
+    )
+    professionalSubjects = screen.getAllByLabelText(
+      /editProfilePage.profile.professionalTab.subject/
+    )
+    submitButton = screen.getByText(
+      /editProfilePage.profile.professionalTab.addCategoryModal.submitBtn/
+    )
```

Comment:
Why are you using regex template here? There is no regex in this strings, so replace them with simple strings

Reply:
I made it according to another tests in this unit test